build_tools/docker/nvidia/Dockerfile

```diff
@@ -26,6 +26,15 @@ RUN wget -q "https://storage.googleapis.com/iree-shared-files/${NVIDIA_COMMON_DE
 RUN wget -q "https://storage.googleapis.com/iree-shared-files/${NVIDIA_GL_DEB?}"
 RUN wget -q "https://storage.googleapis.com/iree-shared-files/${NVIDIA_COMPUTE_DEB}"
 
+# Get the SDK from nvidia's repository
+RUN wget -q https://developer.download.nvidia.com/compute/cuda/repos/debian11/x86_64/cuda-cccl-11-6_11.6.55-1_amd64.deb
```

Comment:
😲 we really need all of these?

Reply:
nvcc-cuda depends on all those unfortunately

Reply:
changed it to a bigger package but that's the only thing I can make work with cmake